README.md

```diff
@@ -783,13 +783,21 @@ easy and simple.
 
 ### Long-term support
 
-You **must** always use the **LTS** (Long-term support) version of Node.js as it is considered
+You **must** always use the **Active LTS** (Long-term support) version of Node.js as it is considered
 stable and will ensure that you don't encounter any unexpected issues. Furthermore, when creating a
-new project or tool, it **must** always target the **LTS** version, unless there is a good reason
+new project or tool, it **must** always target the **Active LTS** version, unless there is a good reason
 not to e.g. an experimental tool or long-term project. To find out the current LTS version, you can
 use a tool such as [nvm](https://github.com/nvm-sh/nvm) or simply check the Node.js
```

Comment:
I think we should recommend [VoltaJs](https://github.com/volta-cli/volta). Some devs are using Windows, nvm-sh doesn't work for them, VoltaJS works great on MacOS and Windows (Presumably also on other operating systems, but haven't used it on anything other than Windows an MacOS).

I think we should discourage people to install NodeJS without a tool like nvm/VoltaJS because it'll make switching projects more difficult.

Reply:
unrelated to the PR, please create a separate issue for this discussion